src/components/ApplyFiltersModal/ApplyFiltersModal.stories.tsx

```diff
@@ -0,0 +1,94 @@
+import { useEffect, useState } from "react";
+import { InMemoryCache } from "@apollo/client";
+import { MockedProvider } from "@apollo/client/testing";
+import { StoryObj } from "@storybook/react";
+import { LogTypes } from "constants/enums";
+import { useLogContext } from "context/LogContext";
+import { TaskQuery, TaskQueryVariables } from "gql/generated/types";
+import { GET_TASK } from "gql/queries";
+import { useQueryParams } from "hooks/useQueryParam";
+import { defaultFiltersMock, noFiltersMock } from "test_data/defaultFilters";
+import ApplyFiltersModal from ".";
+
+const cache = new InMemoryCache({});
+
+export default {
+  component: ApplyFiltersModal,
+};
+
+const Component = ({ ...args }) => {
+  const [, setSearchParams] = useQueryParams();
+  const { setLogMetadata } = useLogContext();
+  const [open, setOpen] = useState(false);
+
+  useEffect(() => {
+    cache.reset();
+    cache.writeQuery<TaskQuery, TaskQueryVariables>({
```

Comment:
This can be replaced with a simple mock apollo context if you use the useQuery method below.